### PR TITLE
Updates for quay.sh

### PR DESCRIPTION
**Pull request by lpichler**
- reflect Mac OS operating system to fix image tag generation
- possibility to switch between pod managers: `docker` or `podman` - for Mac is recommended docker
   -  `env IMAGE_BUILDER=docker ../scripts/clowder/quay.sh sources-api`
   -  `env IMAGE_BUILDER=podman ../scripts/clowder/quay.sh sources-api`
- possibility to build image without cache:
  -  `env CACHE=no-cache ../scripts/clowder/quay.sh sources-api`

cc @slemrmartin

### CODE REVIEW

File: scripts/clowder/quay.sh [hunk 1]
```diff
@@ -7,16 +7,56 @@ if [[ -z $QUAY_ROOT ]]; then
   exit
 fi
 
+if [[ "$(uname)" == "Darwin" ]]; then
+  MAC_OS=true
+else
+  MAC_OS=false
+fi
+
+PODMANAGER="podman"
```
Comment (slemrmartin) [inline scripts/clowder/quay.sh]: I'm maybe for some other name, like IMG_MANAGER or IMAGE_BUILDER etc.

File: scripts/clowder/quay.sh [hunk 2]
```diff
@@ -7,16 +7,56 @@ if [[ -z $QUAY_ROOT ]]; then
...
+
+if [[ -n $2 ]]; then
+  PODMANAGER=$2
```
Comment (slemrmartin) [inline scripts/clowder/quay.sh]: ENV variable will be better than 2nd param (or argument like `--builder=docker` but it's more complicated)

so `env IMAGE_BUILDER=docker ./quay.sh sources-api` will look better

File: scripts/clowder/quay.sh [hunk 3]
```diff
@@ -7,16 +7,56 @@ if [[ -z $QUAY_ROOT ]]; then
...
+    echo "Second parameter have to be podman or docker."
+    exit 2
+  fi
+fi
+
+echo "Used pod manager: $PODMANAGER"
+
+CACHE=true
+
+if [[ $3 == "no-cache" ]]; then
```
Comment (slemrmartin) [inline scripts/clowder/quay.sh]: the same here, consider either named argument like `--no-cache=true` or env variable

**Review by slemrmartin — CHANGES_REQUESTED** [review]

File: scripts/clowder/quay.sh (hunk 1 above)
Comment (lpichler) [reply scripts/clowder/quay.sh]: done

File: scripts/clowder/quay.sh (hunk 2 above)
Comment (lpichler) [reply scripts/clowder/quay.sh]: 👍

File: scripts/clowder/quay.sh (hunk 3 above)
Comment (lpichler) [reply scripts/clowder/quay.sh]: 👍

File: scripts/clowder/quay.sh [hunk 4]
```diff
@@ -45,7 +45,7 @@ image_tag=`eval "$IMAGE_COMMAND"`
 if $CACHE; then
   $IMAGE_BUILDER build -t ${QUAY_ROOT}/${svc} .
 else
-  if [[ IMAGE_BUILDER == "podman" ]]; then
+  if [[ $IMAGE_BUILDER == "podman" ]]; then
```
Comment (lpichler) [inline scripts/clowder/quay.sh:48]: oh I forgot 💰 :)

**Comment by lpichler** [comment]
Done, thanks for review 👍

**Review by slemrmartin — CHANGES_REQUESTED** [review]
great, thanks! just a last little thing, update the README according to the changes

**Comment by lpichler** [comment]
> great, thanks! just a last little thing, update the README according to the changes

 good eye 👀 fixed :+1, thanks 👍

**Review by slemrmartin — APPROVED** [review]

**Pull request merged**